src/mumble/mumble.pro

```diff
@@ -210,12 +210,12 @@ SOURCES *= BanEditor.cpp \
     VoiceRecorderDialog.cpp \
     WebFetch.cpp \
     MumbleApplication.cpp \
-    smallft.cpp \
+    ../../3rdparty/smallft-src/smallft.cpp \
     ThemeInfo.cpp \
     Themes.cpp \
     OverlayPositionableItem.cpp
 
-DIST		*= ../../icons/mumble.ico licenses.h smallft.h ../../icons/mumble.xpm murmur_pch.h mumble.plist
```

Comment:
Nope.

It was used to add extra distfiles for "make dist".
But we haven't used make dist in a long time.
We use scripts/release.pl instead.
And that no longer reads DIST from pro files. It works from the assumption that everything that isn't explicitly blacklisted is in.

So yes, should be able to remove 'em. And I think we should. :-)